llvm/lib/Support/Z3Solver.cpp

```diff
@@ -932,7 +932,8 @@ class Z3Statistics final : public SMTSolverStatistics {
   };
   unsigned getUnsigned(StringRef Key) const override {
     auto It = UnsignedValues.find(Key.str());
-    assert(It != UnsignedValues.end());
+    if (It == UnsignedValues.end())
+      return 0;
```

Comment:
Why did this assertion fail? I thought the "rlimit" must exist on the required Z3 versions.